src/sidebar/services/frame-sync.js

```diff
@@ -76,16 +75,18 @@ export class FrameSyncService {
     /**
      * Channel for sidebar-host communication.
      *
-     * @type {Bridge<SidebarToHostEvent,HostToSidebarEvent>}
+     * @type {PortRPC<HostToSidebarEvent, SidebarToHostEvent>}
      */
-    this._hostRPC = new Bridge();
+    this._hostRPC = new PortRPC();
 
     /**
      * Channel for sidebar-guest(s) communication.
      *
-     * @type {Bridge<SidebarToGuestEvent,GuestToSidebarEvent>}
+     * TODO - Make this a Map<frame ID, PortRPC<...>>?
+     *
+     * @type {PortRPC<GuestToSidebarEvent, SidebarToGuestEvent>[]}
      */
-    this._guestRPC = new Bridge();
+    this._guestRPC = [];
```

Comment:
Similar to the corresponding field in the `Sidebar` class, it will probably make sense to make this a `Map<frame ID, PortRPC>` in future. It is an array for now because we don't have a frame ID at the point where we want to add the PortRPC to this map.